<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>apply</class>
 <widget class="QDialog" name="apply">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Apply</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{background-image: url(:/img/img/inside.jpg);}
QMessageBox{
    font: 87 10pt &quot;Segoe UI Black&quot;;
}
QPushButton:pressed{
	background-color: rgb(255, 212, 39);
   border-radius:6px;
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>Date of Birth</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit">
      <property name="styleSheet">
       <string notr="true">background:black;
color:white;
font: 87 10pt &quot;Segoe UI Black&quot;;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 10pt &quot;Segoe UI Black&quot;;
text-decoration: underline;
font:87 15pt &quot;Segoe UI Black&quot;;</string>
   </property>
   <property name="text">
    <string>Apply for a Tution</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>460</y>
     <width>608</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="styleSheet">
       <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>English Medium</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="styleSheet">
       <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>National Curriculam</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_apply">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>610</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background:black;
color:white;
font: 87 13pt &quot;Segoe UI Black&quot;;
border:black;
}
QPushButton:pressed{
	
	background-color: rgb(182, 11, 42);
   border-radius:6px;
}
QPushButton:hover{
	
	background-color: grey;
   border-radius:6px;
}</string>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/img/img/icon.png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>510</y>
     <width>339</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="styleSheet">
       <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>Male</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="styleSheet">
       <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>Female</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>331</width>
     <height>238</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_name">
        <property name="styleSheet">
         <string notr="true">*{
background:black;
color:white;
font: 87 15pt &quot;Segoe UI Black&quot;;
border:black;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_rad">
        <property name="styleSheet">
         <string notr="true">*{
background:black;
color:white;
font: 87 15pt &quot;Segoe UI Black&quot;;
border:black;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_group">
        <property name="styleSheet">
         <string notr="true">*{
background:black;
color:white;
font: 87 15pt &quot;Segoe UI Black&quot;;
border:black;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_email">
        <property name="styleSheet">
         <string notr="true">*{
background:black;
color:white;
font: 87 15pt &quot;Segoe UI Black&quot;;
border:black;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_rec">
      <property name="styleSheet">
       <string notr="true">*{
background:black;
color:white;
font: 87 15pt &quot;Segoe UI Black&quot;;
border:black;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>138</width>
     <height>228</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_name">
        <property name="styleSheet">
         <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_rad">
        <property name="styleSheet">
         <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
        </property>
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_group">
        <property name="styleSheet">
         <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
        </property>
        <property name="text">
         <string>Group</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_email">
        <property name="styleSheet">
         <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_rec">
      <property name="styleSheet">
       <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>Recruiter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>270</y>
     <width>331</width>
     <height>190</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_phone">
        <property name="styleSheet">
         <string notr="true">*{
background:black;
color:white;
font: 87 15pt &quot;Segoe UI Black&quot;;
border:black;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_inst">
        <property name="styleSheet">
         <string notr="true">*{
background:black;
color:white;
font: 87 15pt &quot;Segoe UI Black&quot;;
border:black;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_class">
        <property name="styleSheet">
         <string notr="true">*{
background:black;
color:white;
font: 87 15pt &quot;Segoe UI Black&quot;;
border:black;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_ac">
      <property name="styleSheet">
       <string notr="true">*{
background:black;
color:white;
font: 87 15pt &quot;Segoe UI Black&quot;;
border:black;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>270</y>
     <width>174</width>
     <height>182</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_phone">
        <property name="styleSheet">
         <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
        </property>
        <property name="text">
         <string>Phone</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_ins">
        <property name="styleSheet">
         <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
        </property>
        <property name="text">
         <string>Institution</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_class">
        <property name="styleSheet">
         <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
        </property>
        <property name="text">
         <string>Class/Dept.</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_ac">
      <property name="styleSheet">
       <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
      </property>
      <property name="text">
       <string>Apply Code</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="bbn">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>0</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{image: url(:/img/img/backbutoonyblack.jpg);
border:white}
QPushButton:pressed{
	
	background-color: rgb(182, 11, 42);
   border-radius:6px;
}
QPushButton:hover{
	
	background-color: rgb(182, 11, 42);
   border-radius:6px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
